### PR TITLE
Cluster threshold parameter in dipy_buan_shapes workflow

### PR DESCRIPTION
**Pull request by BramshQamar**
Added cluster threshold (clust_thr) as an optional argument in dipy_buan_shapes workflow. 
Updated default clust_thr values as well.

This small update makes the workflow faster.

### CODE REVIEW

File: dipy/segment/bundles.py [hunk 1]
```diff
@@ -162,7 +163,9 @@ def bundle_shape_similarity(bundle1, bundle2, rng, threshold=5):
     bundle2 : Streamlines
         White matter tract from another subject (eg: AF_L)
     rng : RandomState
-    threshold : float (default 5)
+    clust_thr : float (default [5, 3, 1.5])
```
Comment (skoudoro) [inline dipy/segment/bundles.py:166]: list of float instead of float

File: dipy/workflows/stats.py [hunk 2]
```diff
@@ -527,9 +528,13 @@ def run(self, subject_folder, threshold=6, out_dir=''):
             Path to the input subject folder. This path may contain
             wildcards to process multiple inputs at once.
 
+        clust_thr : string (default [5,3,1.5]), optional
```
Comment (skoudoro) [inline dipy/workflows/stats.py:531]: `variable float` instead of `string`

File: dipy/workflows/stats.py [hunk 3]
```diff
@@ -543,6 +548,11 @@ def run(self, subject_folder, threshold=6, out_dir=''):
 
         """
         rng = np.random.RandomState()
+
+        clust_thr = list(map(float, clust_thr.strip('[]').split(',')))
+        #clust_thr=[5,3,1.5]
+        print(clust_thr)
```
Comment (skoudoro) [inline dipy/workflows/stats.py:554]: remove the print

File: dipy/workflows/stats.py [hunk 4]
```diff
@@ -543,6 +548,11 @@ def run(self, subject_folder, threshold=6, out_dir=''):
 
         """
         rng = np.random.RandomState()
+
+        clust_thr = list(map(float, clust_thr.strip('[]').split(',')))
```
Comment (skoudoro) [inline dipy/workflows/stats.py:552]: you do not need that if you use `variable float`

Comment (BramshQamar) [reply dipy/workflows/stats.py:552]: but it is a list
if I just write float
it gives error

Comment (BramshQamar) [reply dipy/workflows/stats.py:552]: let me try one more time

Comment (skoudoro) [reply dipy/workflows/stats.py:552]: `variable float` not just `float` and you will get your list.

File: dipy/workflows/stats.py (hunk 2 above)
Comment (BramshQamar) [reply dipy/workflows/stats.py:531]: should I call workflow like this:
`dipy_buan_shapes subjects_small/ --clust_thr [5,3,1.25] --out_dir "sm_test"`

or

`dipy_buan_shapes subjects_small/ --clust_thr "[5,3,1.25]" --out_dir "sm_test"`

both are giving me an error

Comment (BramshQamar) [reply dipy/workflows/stats.py:531]: got it, like this:

`dipy_buan_shapes subjects_small/ --clust_thr 5 3 1.25 --out_dir "sm_test"`

Thank you @skoudoro

**Comment by Garyfallidis** [comment]
Thank you @BramshQamar !

**Pull request merged**